### DISCUSSION TITLE
Bevallás riport visszakérése az API-n keresztül

### DISCUSSION THREAD

**Discussion by Edi1978** [nav-gov-hu/eVAT#11] [Kérdések és válaszok - Gépi interfész] [answered]
A validálásra feltöltött eÁfa M2M XML api üzenet után is visszakérhető-e a bevallás riport (összesítő) (Jelenlegi bevallás XML)? Ha igen, mennyi idő elteltével? Ebben az állapotban még csak validáláson esett át, és tegyük fel, hogy nincs ERROR válasz. Ekkor lekérhető már a bevallás riport? Vagy csak akkor, ha már bevallásként elfogadásra is került az XML?

Az is kérdés, hogy a Manage declaration API üzenet beküldése után mennyi idő után tudom lekérni a NAV-tól a bevallás XML-t?

**Comment by NTCA-tax** [comment, marked as answer]
Jelenleg mi abban gondolkodtunk, hogy a beadott bevallásból kérhető vissza egy bevallás riport. Látnád annak előnyeit, hogy a már feltöltött, de be nem adott eÁfa M2M XML-ből is adjunk vissza bevallás riport összesítőt?

A Manage declaration API beküldését követően elkezdődik az adóhivatali oldalon a bevallás feldolgozása. Miután ez megtörtént, akkor kérhető vissza a bevallás riport. Mivel még nincs kész rendszerünk, hanem tervezeti szinten vagyunk, így átlagos időt nem tudok mondani, de nagy eséllyel nem lesz több néhány percnél.

**Reply by Edi1978** [reply]
Igen, mi egyértelműen látnánk az előnyeit. Az ügyfél biztosan látni szeretné akár tört időszakban, akár az XML bevallásként való elfogadása előtt a riport összesítőt. Mi nagyon örülnénk az ezirányú fejlesztésnek!
